pytorch_lightning/plugins/training_type/fully_sharded.py

```diff
@@ -178,6 +182,19 @@ def lightning_module_state_dict(self) -> Dict[str, Union[Any, Tensor]]:
         # state dict.
         return super().lightning_module_state_dict()
 
+    def serialized_restore_model_state(self, checkpoint_path: Union[str, Path]) -> Dict[str, Any]:
+        checkpoint = {}
+        log.info(f"FullyShardedDataParallel has {self.num_processes} processes. Serializing to avoid CPU OOMs.")
```

Comment:
we probably don't want to print that several times right?
so we could use the function `rank_zero_info` from the utilities.

Reply:
thanks for the suggestion; what a clever method!